pytorch_lightning/tuner/lr_finder.py

```diff
@@ -60,6 +63,174 @@ def _determine_lr_attr_name(trainer: 'pl.Trainer', model: 'pl.LightningModule')
     )
 
 
+def lr_find(
```

Comment:
could you move it to the old place. easier for us to review the changes :)

Reply:
done @awaelchli